cmd/common.go

```diff
@@ -129,7 +129,7 @@ func syncMain(filenames []string, dry bool, parallelism, delay int, workspace st
 		}
 
```

Comment:
@mflendrich You usually have good testing tricks up your sleeve. Is there a way to test this **without** spinning up a Kong instance and **without** significant mocking overhead? I might be asking for too much.

Thinking out loud, we should be able to refactor and unit test large parts of this component.
However, that would be a separate task and not part of this PR.
Leaving this comment to have a brief discussion while we are here.

Reply:
> without spinning up a Kong instance and without significant mocking overhead?

Without a Kong instance I cannot think of a test here that wouldn't be a "change detector" test.

> we should be able to refactor and unit test large parts of this component

For logic (like, translating something to something else) I'm all in for unit tests, however for integrations (like, "action X results in effect Y") I'd opt for tests featuring a real but minimal Kong. What makes it complex is that such a Kong would need to be a part of the test harness, now that we don't have a way to spin up a clean fresh small instance of Kong programmatically.